actions/run/src/run.ts

```diff
@@ -46,10 +47,12 @@ export async function run(): Promise<void> {
   for (const t of targetsArgs) {
     core.info(`Running target: ${t}`)
     const argsSpawn = [...args]
-    // Artifact is set
-    if (artifact) {
-      core.info(`Pushing target ${t} with artifact tag`)
+
+    if (artifactSaveLocal) {
       argsSpawn.push('--artifact', `${t}/`, `${artifactPath}`)
+    } else if (artifactName.trim() !== '') {
+      core.info(`Pushing target ${t} with artifact tag`)
```

Comment:
This is a confusing log entry. Pushing makes sense when you read the code, but looking through the logs it will be unclear what "pushing target" means (could easily be confused with pushing a container image).

Reply:
Will change to  `Running --artifact with target ${t}, output to ${artifactPath}`

Reply:
I changed to `Running --artifact with target ${t}, output to ${artifactPath}`